test/integration/machine-learning.spec.ts

```diff
@@ -45,5 +163,44 @@ describe('admin.machineLearning', () => {
       .should.eventually.be.rejected.and.have.property(
         'code', 'machine-learning/invalid-argument');
     });
+
+    it('deletes existing Model', () => {
+      return createTemporaryModel().then((model) => {
+        return admin.machineLearning().deleteModel(model.modelId)
+          .then(() => {
+            return admin.machineLearning().getModel(model.modelId)
+              .should.eventually.be.rejected.and.have.property('code', 'machine-learning/not-found');
+          })
+          .then(() => {
+            unscheduleForDelete(model);  // Already deleted.
+          });
+      });
+    });
   });
+
+  function verifyModel(model: admin.machineLearning.Model, modelOptions: admin.machineLearning.ModelOptions) {
+    expect(model.displayName).to.equal(modelOptions.displayName);
+    expect(model.createTime).to.not.be.empty;
+    expect(model.updateTime).to.not.be.empty;
+    expect(model.etag).to.not.be.empty;
+    if (modelOptions.tags) {
+      expect(model.tags).to.deep.equal(modelOptions.tags);
+    } else {
+      expect(model.tags).to.be.empty;
+    }
+    if (modelOptions.tfliteModel) {
+      expect(model.tfliteModel!.gcsTfliteUri).to.equal(modelOptions.tfliteModel.gcsTfliteUri);
```

Comment:
Nit: Refactor this portion into a separate helper.

Reply:
done